_ext/eclipse-base/gradle.properties

```diff
@@ -1,7 +1,7 @@
 # Mayor versions correspond to the supported Eclipse core version.
 # Minor version is incremented for features or incompatible changes (including changes to supported dependency versions).
 # Patch version is incremented for backward compatible patches of this library.
-ext_version=3.2.1
```

Comment:
If we don't change the code in `eclipse-base`, then we don't need to publish a new `eclipse-base`.  We only change this when something about the OSGi plumbing needs to change, which is generally rare.

Reply:
I was under the impression that the compile-scoped dependencies were included in the fat jar, but if that is incorrect, then I can back this and the related changes out.
Are you OK with me rewriting these commits to suit, or would you prefer new commits?